test/user-profile/components/FormList.unit.spec.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import FormList from '../../../src/js/user-profile/components/FormList.jsx';
+
+describe('<FormList>', () => {
+  const props = {
+    savedForms: [
+      {
+        form: '1010ez',
+        metadata: {
+          last_updated: '1503688891',  // eslint-disable-line camelcase
+          expires_at: '1503688891'  // eslint-disable-line camelcase
+        }
+      }
+    ]
+  };
+  let windowOpen;
+  let oldWindow;
+  const setup = () => {
```

Comment:
Can we pull this out into a helper file so we don't have to duplicate this boilerplate elsewhere?

We'd just want to make it customizable on what the stub returns when you call it, but that should be easy.